skopt/gp_opt.py

```diff
@@ -95,7 +96,9 @@ def gp_minimize(func, dimensions, base_estimator=None, acq="LCB", xi=0.01,
         Number of calls to `func`.
         If `n_random_starts` > 0, `n_calls - n_random_starts`
         additional evaluations of `func` are made that are guided
-        by the `base_estimator`.
+        by the `base_estimator`. If `x0` is provided but not `y0`
+        then `n_calls - len(x0) - n_random_starts` evaluations
+        are made instead of `n_calls - n_random_starts` .
```

Comment:
Just after dimension or right at the top, before func ? as the ordering of the parameters on the signature should correspond to the one on the doc, after dimesions I can switch ordering in the signature as well, but it would not be possible before func because they are not optional.